spec/features/create_review_spec.rb

```diff
@@ -22,6 +27,31 @@
       expect(page).to have_content("2")
     end
 
+    scenario "user properly enters a new review" , %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Space found before comma.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.